packages/apollo/src/utils/getOperationManifestFromProject.ts

```diff
@@ -13,14 +13,16 @@ export interface ManifestEntry {
 }
 
 export function getOperationManifestFromProject(
-  project: GraphQLClientProject
+  project: GraphQLClientProject,
+  options: { stripLiterals: boolean } = { stripLiterals: true }
```

Comment:
```suggestion
  options: {
    preserveStringAndNumericLiterals: boolean,
  } = {
    preserveStringAndNumericLiterals: true,
  }
```

Reply:
Presumably, the default should also flip to false? :)